01-ruby-fundamentals/06-oo-methods.md

```diff
@@ -96,3 +96,26 @@ puts result
 ```
 
 **Methods can often be referred to as _messages_. When we send a message, we expect a reply.**
+
+## Method Vocabulary
+
+This vocab list will reference the following code:
+
+```ruby
+def sum(a, b, c)
+  result = a + b + c
+  return result
+end
+
+number = sum(5, 17, 106)
+puts number
+```
+
+Term             | Definition | Example&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Good - I think some of the "terms" by themselves are a bit odd, but paired with the definition it works for me.